plugin/sqlsh/src/main/java/com/twosigma/beaker/sqlsh/utils/ConnectionStringHolder.java

```diff
@@ -0,0 +1,120 @@
+package com.twosigma.beaker.sqlsh.utils;
```

Comment:
What do You mean ?

Does this ? If Yes , then what year to put ? 2016 ?
Copyright 2014 TWO SIGMA OPEN SOURCE, LLC ....

For now put this "Copyright" as it was.